README.md

```diff
@@ -4,6 +4,16 @@
 
 This application connects tags from a Flexy to an OSIsoft PI dataserver.
 
+## Support
+
+This application is supported by HMS' North American offices.
+
+| Phone                                        | Forum                                      | Email                                           |
+|:--------------------------------------------:|:------------------------------------------:|:-----------------------------------------------:|
+| [+1 312 829 0601](tel:13128290601), Option 2 | [hms.how](https://forum.hms-networks.com/) | [us-services@hms.se](mailto:us-services@hms.se) |
+
```

Comment:
did you mean to change option 2 to a phone number?

Reply:
No, its option #2 for support when you dial in

Reply:
https://www.anybus.com/support/contact-support

this is the format we use on the support page

Reply:
do you call the number and select option 2 in some menu? or is it saying that the phone number is option 2?

Reply:
It is option 2 in the menu